src/Migration/OldTaskFinder.php

```diff
@@ -36,16 +35,17 @@ public function all(?string $plugin) {
 	 * @return array<string>
 	 */
 	protected function getTasks(string $path, ?string $plugin) {
-		$Folder = new Folder($path);
-		$res = $Folder->find('Queue.+Task\.php');
+		$res = glob($path . '*Task.php');
 
 		$tasks = [];
-		foreach ($res as $key => $r) {
-			$name = basename($r, 'Task.php');
-			$name = substr($name, 5);
-
-			$taskKey = $plugin ? $plugin . '.' . $name : $name;
-			$tasks[$taskKey] = $path . $r;
+		if($res) {
```

Comment:
cs issues

Reply:
Had to add that if there because `glob()` can return `false` on error.
But I will more restrictly check if that result is not false.

Reply:
talking about whitespace

Reply:
ahhhhh, but then why is there no cs-rule present for that?

Reply:
I am sure there is, you just didnt execute it.

Reply:
just saying, I executed `composer cs-fix` and there was nothing changed so there is nothing wrong (with no space between if and the `(`)

Reply:
but for some reason `composer cs-check` does report it